### PR TITLE
[Documentation] Expanded demistomock docstring to emphasise incidents keys

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Description
Added docstring detailing the fields that are expected to be in incidents when calling `demisto.createIncidents`, with emphasis on `type` - as it's not required by the server, but should be provided.

### CODE REVIEW

File: Tests/demistomock/demistomock.py [hunk 1]
```diff
@@ -859,7 +859,15 @@ def createIncidents(incidents, lastRun=None, userID=None):
     Creates incident in long running execution
 
     Args:
-      incidents (list): List of incident objects to create
+      incidents (list):
+        List of incident objects to create, with the following required keys, and some common optional keys
+            - name (required) - str
+            - type (required) - if not provided will create unclassified incident) - str
```
Comment (yaakovi) [inline Tests/demistomock/demistomock.py:865]: so not required?

File: Tests/demistomock/demistomock.py [hunk 2]
```diff
@@ -859,7 +859,15 @@ def createIncidents(incidents, lastRun=None, userID=None):
...
+            - labels (required) - list of {"type": _, "value": _} objects
```
Comment (yaakovi) [inline Tests/demistomock/demistomock.py:866]: are you sure labels are required?

File: Tests/demistomock/demistomock.py (hunk 1 above)
Comment (DeanArbel) [reply Tests/demistomock/demistomock.py:865]: it's not required by the server, but we don't want to create incidents that are unclassified, so it's "required" 🙂

File: Tests/demistomock/demistomock.py (hunk 2 above)
Comment (DeanArbel) [reply Tests/demistomock/demistomock.py:866]: You're right, they are not required. fixing via suggestion:
```suggestion
            - labels (optonal) - list of {"type": _, "value": _} objects
```

File: Tests/demistomock/demistomock.py (hunk 1 above)
Comment (DeanArbel) [reply Tests/demistomock/demistomock.py:865]:
```suggestion
            - type (required - if not provided will create unclassified incident) - str
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**